### PR TITLE
Smaato: Split multiple media types

### PR DESCRIPTION
**Pull request by el-chuck**
Split multiple media types of impressions into multiple requests

### CODE REVIEW

File: adapters/smaato/smaato.go [hunk 1]
```diff
@@ -160,24 +160,53 @@ func (adapter *adapter) makeIndividualRequests(request *openrtb2.BidRequest) ([]
...
 
 	for _, imp := range imps {
-		request.Imp = []openrtb2.Imp{imp}
-		if err := prepareIndividualRequest(request); err != nil {
-			errors = append(errors, err)
-			continue
-		}
-
-		requestData, err := adapter.makeRequest(request)
+		impsByMediaType, err := splitImpressionsByMediaType(&imp)
```
Comment (Enigo) [inline adapters/smaato/smaato.go:163]: okay, Im completely puzzled O_O
you added `bidfloor` to json but I don't see it anywhere in the adapter code.
how does it work?

Comment (el-chuck) [reply adapters/smaato/smaato.go:163]: it is already openrtb, so when request with imp[].bidfloor hits auction endpoint we just pass it along

Comment (Enigo) [reply adapters/smaato/smaato.go:163]: gotcha, thx!

**Comment by Enigo** [comment]
LGTM

**Pull request closed without merging**